petri/src/vm/hyperv/vm.rs

```diff
@@ -248,23 +256,22 @@ impl HyperVVM {
     pub async fn start(&self) -> anyhow::Result<()> {
         self.check_state(VmState::Off)?;
         hvc::hvc_start(&self.vmid)?;
-        self.wait_for_state(VmState::Running).await
```

Comment:
Why are these removed?

Reply:
this test actually breaks because the test completes inbetween the start and wait for state. Trevor and I chatted a bit last week and he said to remove it since it's unneeded - the start/other commands should fail so we don't need the check.

Reply:
OK